code/graphics/opengl/gropengldeferred.cpp

```diff
@@ -433,6 +433,7 @@ void gr_opengl_deferred_light_cylinder_init(int segments) // Generate a VBO of a
 	// make sure we have one
 	if (deferred_light_cylinder_vbo) {
 		glBindBuffer(GL_ARRAY_BUFFER, deferred_light_cylinder_vbo);
+		glBufferData(GL_ARRAY_BUFFER, nVertex * sizeof(float), nullptr, GL_STATIC_DRAW);
```

Comment:
Can you explain why you added this? The initialized data will be overwritten by the call in the next line so this call is not necessary.